src/flamenco/runtime/context/fd_exec_txn_ctx.c

```diff
@@ -151,11 +152,14 @@ fd_txn_borrowed_account_modify_idx( fd_exec_txn_ctx_t *        ctx,
   }
 
   fd_borrowed_account_t * txn_account = &ctx->borrowed_accounts[idx];
+  if( !fd_txn_account_is_writable_idx( ctx, (int)idx ) ) {
```

Comment:
UNLIKELY.

Reply:
I don't love how we handle the borrowed account modify semantics... Probably should be addressed in a different PR